tests/test__socket_errors.py

```diff
@@ -54,9 +55,9 @@ def test_timeout(self, socket=socket):
 
 
 def test_create_connection_refused():
-    errno = None
     try:
         socket.create_connection(('127.0.0.1', 1))
     except socket.error as ex:
-        errno = ex.errno
-    assert errno in [111, 61, 10061], 'Expected socket.error ECONNREFUSED, got {0}'.format(errno)
+        code, text = ex.args
+        self.assertEqual(code, errno.ECONNREFUSED,
```

Comment:
Let's keep the previous indentation level here I think, instead this test can silently fail if `socket.create_connection()` succeeds for some reason

Reply:
Yes, I missed that case, sorry.

I think it's better to be consistent with the above code in handling it, though, and add one more assert before `except`. I'll update the pull request.